vra-services-check/src/__init__.py

```diff
@@ -0,0 +1,25 @@
+""" 
+Copyright © 2018 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the “License”); you may not
```

Comment:
This is using the Apache license.  I think you mean to be using the MIT license as you have elsewhere.

Reply:
Yes, will use The MIT license (the “License”)